pkg/commands/update/check_test.go

```diff
@@ -136,7 +137,16 @@ func TestCheckAsync(t *testing.T) {
 				out bytes.Buffer
 				buf bytes.Buffer
 			)
-			f := update.CheckAsync(ctx, testcase.file, configFilePath, testcase.currentVersion, testcase.cliVersioner, in, &out)
+			f := update.CheckAsync(
+				ctx,
+				testcase.file,
+				configFilePath,
+				testcase.currentVersion,
+				testcase.cliVersioner,
+				in,
+				&out,
+				fsterr.MockLog{},
```

Comment:
Passing through a mock error log instance and multi-lining this method call for readability.